app/scripts/controllers/download.js

```diff
@@ -54,10 +55,17 @@ angular.module('lantern_www')
       return false;
     };
 
+    $scope.hideBetaLink = false; 
+
+    if ($translate.use() == constants.LANGS.fa_IR.code) {
+        $scope.hideBetaLink = true;
+    };
+
     $scope.trackEvent = function(type) {
         $window.ga('send', 'event', type, 
                    'click', osSniffer.os);
         if (type === 'download' && $translate.use() == constants.LANGS.fa_IR.code) {
+            $scope.irDownload = true;
```

Comment:
Is it too late to set `irDownload` to `true` here? IIUC we need to set it during page rendering.